include/lsst/afw/table/io/python.h

```diff
@@ -41,45 +41,10 @@ namespace table {
 namespace io {
 namespace python {
 
-/**
- * Wraps an instantiation of @ref PersistableFacade.
- *
- * @deprecated Use addPersistableMethods for all new code.
- *
- * Pybind11 shall assume that `PersistableFacade` is managed using
- * `std::shared_ptr`, as this is required for compatibility with
- * existing subclasses of `PersistableFacade`. This means that wrapping
- * will only work if new classes also use `std::shared_ptr` as their
- * holder type.
- *
- * @tparam T The type of object this `PersistableFacade` is for.
- *
- * @param module The pybind11 module that shall contain `PersistableFacade<T>`
- * @param suffix A string to disambiguate this class from other
- *               `PersistableFacades`. The Python name of this class shall be
- *               `PersistableFacade<suffix>`.
- */
-template <typename T>
-[[deprecated("Use `addPersistableMethods` instead. To be removed after 20.0.0.")]]  // DM-22814
-        void
-        declarePersistableFacade(pybind11::module &module, std::string const &suffix) {
-    using namespace pybind11::literals;
-
-    pybind11::class_<PersistableFacade<T>, std::shared_ptr<PersistableFacade<T>>> cls(
-            module, ("PersistableFacade" + suffix).c_str());
-    cls.def_static("readFits",
-                   (std::shared_ptr<T>(*)(std::string const &, int)) & PersistableFacade<T>::readFits,
-                   "fileName"_a, "hdu"_a = fits::DEFAULT_HDU);
-    cls.def_static("readFits",
-                   (std::shared_ptr<T>(*)(fits::MemFileManager &, int)) & PersistableFacade<T>::readFits,
-                   "manager"_a, "hdu"_a = fits::DEFAULT_HDU);
-}
-
 /**
  * Add table::io::Persistable and PersistableFacade methods to the pybind11 wrapper for a class
  *
- * Use this instead of declarePersistableFacade to avoid circular import issues in Python;
- * it allows your class to be used without importing lsst.afw.table.
+ * This allows your class to be used without importing lsst.afw.table.
```

Comment:
Without the previous statement this sentence isn't very clear. Maybe append "in Python", to distinguish from imports needed inside pybind11 code?